### PR TITLE
Edit new joiners guide

### LINKED ISSUE #44: Add a description of emoji voting
## Summary
Add a bit describing how we vote for projects using emojis. Including things like the fact that you can thumbup/smiley because you want to learn something you don't already know.

This could be in the new starter pages, or another "how we work" section.

## Impact
Handy to link to for new starters. Also a nice advertisement if e.g. prospective recruits browse the handbook, makes us look desirable (because we are!). Project voting is a pretty integral part of how REG functions.

### Updates
@mhauru and @fedenanni discussed this on 2022-08-09, Fede said he might write this, just making an issue to track.

@mhauru, @fedenanni and @dsj976 discussed this on 2022-08-25 and decided that @dsj976 would take care of updating this in the manual.

### LINKED ISSUE #45: Public Profile on Turing Website
## Summary
The New Joiners Guide (in Systems Set Up) currently says that to create a public profile on the Turing website you should do this through Turing Complete.
This is no longer the case and should be done by emailing the [Web team](mailto:website@turing.ac.uk) with a bio description and attached profile picture.

### PR DESCRIPTION
**Pull request by dsj976**
## Summary
This is an update to the REG handbook based on @dsj976 's first couple of weeks at REG - will hopefully clarify a few things to future new joiners

## Related issues
- Closes #44 
- Closes #45

### CODE REVIEW

File: content/docs/onboarding/new_joiners/systems_set_up.md [hunk 1]
```diff
@@ -160,13 +160,9 @@ This system is used for:
...
 ### Create a Public Profile on the Turing Website
 
-Fill out the form on [Turing Complete](https://turingcomplete.topdesk.net/tas/public/ssp/): HOME > COMMUNICATIONS AND EVENTS > WEBSITE AND SOCIAL MEDIA > NEW RESEARCHER PROFILE FOR WEBSITE
-
-Remember to indicate that you are part of REG by selecting "Research Engineering" under Researcher role.
-This way your profile will appear in our group web page.
-
-If you're not in the office there won't have been a headshot photo taken of you.
-If you like, add a photo you've taken of yourself that you'd be happy to see on the website.
+Get in touch with the [Web team](mailto:website@turing.ac.uk).
```
Comment (JimMadge) [inline content/docs/onboarding/new_joiners/systems_set_up.md:163]: Can you add the web team to '[The REGistry](https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry#responsibilities)' under the 'Points of Contact' section and link to that?

We try to consolidate that information and keep emails/identifies off the public site.

File: content/docs/onboarding/new_joiners/systems_set_up.md [hunk 2]
```diff
@@ -160,13 +160,9 @@ This system is used for:
...
+Send them an email with your bio and a profile picture attached.
```
Comment (JimMadge) [inline content/docs/onboarding/new_joiners/systems_set_up.md:164]:
```suggestion
Send them an email with your biography and a profile picture attached.
```

File: content/docs/onboarding/new_joiners/systems_set_up.md [hunk 3]
```diff
@@ -218,6 +214,15 @@ If you want a phone number ask [IT](https://github.com/alan-turing-institute/res
 
 [Mathison](https://mathison.turing.ac.uk/) is the Institute's intranet. A few things still need to be done on the old intranet, [Turing Complete](https://turingcomplete.topdesk.net).
 
+### Purchasing home office equipment
+
+There is a budget to purchase peripherals (monitor, mouse, keyboard, etc.) as well as other equipment such as an office chair and desk.
```
Comment (JimMadge) [inline content/docs/onboarding/new_joiners/systems_set_up.md:219]:
```suggestion
There is a budget to purchase peripherals (monitor, mouse, keyboard, _etc._) as well as other equipment such as an office chair and desk.
```

File: content/docs/onboarding/new_joiners/systems_set_up.md [hunk 4]
```diff
@@ -218,6 +214,15 @@ If you want a phone number ask [IT](https://github.com/alan-turing-institute/res
...
+For peripherals, get in touch with [IT Services](mailto:ITServices@turing.ac.uk).
```
Comment (JimMadge) [inline content/docs/onboarding/new_joiners/systems_set_up.md:220]:
```suggestion
For peripherals, get in touch with [IT Services]({{< ref "https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry#points-of-contact" >}}).
```

**Review by JimMadge — CHANGES_REQUESTED** [review]
Thanks for these additions. They should be very helpful for new joiners in the future.

Just a few changes/suggestions and I think it will be ready to merge.

File: content/docs/onboarding/new_joiners/systems_set_up.md (hunk 1 above)
Comment (dsj976) [reply content/docs/onboarding/new_joiners/systems_set_up.md]: Done

**Comment by JimMadge** [comment]
@dsj976 Sorry, I told you to specify the external links incorrectly. Should be fixed now.

**Review by JimMadge — APPROVED** [review]

**Pull request merged**